crates/linker/Cargo.toml

```diff
@@ -16,6 +16,8 @@ roc_mono = { path = "../compiler/mono" }
 roc_build = { path = "../compiler/build" }
 roc_collections = { path = "../compiler/collections" }
 roc_error_macros = { path = "../error_macros" }
+roc_load = { path = "../compiler/load" }
+roc_reporting = { path = "../reporting" }
```

Comment:
bit unfortunate that the linker depends on all of this. I see it also depends on roc_build already, so this should not influence build times really, but conceptually it feels wrong

Reply:
This will go away once we have surgical linking on all platforms, the deps are only temporary. Not sure what build is used for, maybe we can reorder a call stack to fix that.